### PR TITLE
Pr workflow

### PR DESCRIPTION
**Pull request by bab4379**
## Overview
This PR includes documentation of our Pull Request Reviewer process. 

## Checklist

- [X] Proof-reading
- [X] Any diagrams/figures have their source files included

### CODE REVIEW

File: github/workflow.md [hunk 1]
```diff
@@ -27,7 +27,43 @@ Below is an example topic branch, `edit-screen` that started from the dev branch
...
+  - Does the code produce the expected as described in the draft PR
+
+
+- Developer & Reviewer : If no issues are found the reviewer will approve the pull request.  At this point the developer should merge their changes and close the feature branch.  If the reviewer believes changes are required or has questions for the developer the reviewer should leave comments for the developer.  The developer and reviewer should work together to resolve the reviewers concerns.  If rework is needed, once it is complete the developer should click the Re-request review button on the PR to signal to the reviewer the re-work is ready for review.
+
+![Workflow Diagram](https://github.com/NIT-Administrative-Systems/AS-CloudDocs/blob/8fa5d683bd0b7e2720ed79d4ad77c0fde18140ff/assets/PR%20Workflow.png)
+
+
+### Scenarios
+- My PR has been approved, I have merged and deployed my code but it does not fix the issue.  I continue to add commits to my feature.  Do I need approval again
```
Comment (nie7321) [inline github/workflow.md:64]:
```suggestion
- My PR has been approved, I have merged and deployed my code but it does not fix the issue.  I continue to add commits to my feature.  Do I need approval again?
```

**Review by nie7321 — APPROVED** [review]
suggested one thing from proofreading, otherwise good to go 👍

**Review by nie7321 — APPROVED** [review]

**Review by cecolby11 — APPROVED** [review]
LGTM! 

I thought I remembered a more recent version of the diagram where it said 'communicate with the reviewer' instead of 'Mark as Ready for Review'? not a huge difference, except that marking a PR as ready for review doesn't notify the reviewer

**Comment by bab4379** [comment]
Yes, thank you.  The new version did not get uploaded properly.

**Pull request merged**